components/overlay/index.css

```diff
@@ -28,7 +30,7 @@ governing permissions and limitations under the License.
 
   opacity: 1;
 
-  transition-delay: 0ms;
+  transition-delay: var(--spectrum-animation-duration-0);
```

Comment:
This ties into the same question that Megan had earlier about whether these should be using the same styles. If so, this file would also have the `--spectrum-overlay-animation-duration-opened` (`--spectrum-overlay-animation-duration-0`) defined?